### DISCUSSION TITLE
No compatible libraries for MX Linux

### DISCUSSION THREAD

**Discussion by Barton32164** [phcode-dev/phoenix#1879] [Q&A] [answered]
When I try and download and install Pheonix Code this message is displayed:
No compatible binary found for the current GLIBC version (2.28). Exiting installation.
wbm@mx3020:~
Is there a package compatible with MX?

**Comment by abose** [comment, marked as answer]
Phoenix code requires Linux with atleast glibc version 2.31 or higher. That covers most distros released since 2018. We do not support older distributions and unfortunately we may not support it due to platform limitations.

**Comment by Barton32164** [comment]
Thanks
